modin/core/execution/ray/implementations/pandas_on_ray/partitioning/partition.py

```diff
@@ -199,25 +198,21 @@ def mask(self, row_labels, col_labels):
         self._is_debug(log) and log.debug(f"ENTER::Partition.mask::{self._identity}")
         new_obj = super().mask(row_labels, col_labels)
         if isinstance(row_labels, slice) and isinstance(
-            self._length_cache, ObjectIDType
+            (len_cache := self._length_cache), ObjectIDType
```

Comment:
what's the point of assigning `self._length_cache` to a variable?

Reply:
Because _length_cache is not just a simple field but a property, that may require calculation and new objects allocation. That's why the returned value is cached here.